src/api/java/com/minecolonies/api/compatibility/resourcefulbees/NoBeeCompat.java

```diff
@@ -0,0 +1,19 @@
+package com.minecolonies.api.compatibility.resourcefulbees;
+
+import net.minecraft.item.ItemStack;
+import net.minecraft.item.Items;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class NoBeeCompat implements IBeehiveCompat {

```

Comment:
why do we need this one, can't we use the interface impl just as well as the default?

Reply:
You can't instantiate an interface.

Reply:
you should be able to do a new IBeehiveCompat { }

Reply:
IBeehiveCompat is an interface, which is abstract. As such it's not allowed to contain a constructor and also is not allowed to be instantiated.

Technically the method in IBeehiveCompat should also be completely blank

Reply:
The code that was there previously worked, you can make an anonymous instantiation of the class the way I said it.